packages/curvenote/src/web.ts

```diff
@@ -30,14 +26,7 @@ function makeCurvenoteStartCLI(program: Command) {
 
 function makeBuildCLI(program: Command) {
   const command = new Command('build')
-    .description(
-      'Build pdf, tex, and word exports from MyST files as well as build MyST site content',
-    )
-    .argument('[files...]', 'list of files to export')
-    .addOption(makePdfOption('Build'))
```

Comment:
None of this stuff worked anyway... the function we invoke from this command only does `site` build, no matter what.